rust/src/dcerpc/dcerpc.rs

```diff
@@ -618,6 +620,10 @@ impl DCERPCState {
                     tx.req_lost = true;
                 }
                 tx.req_done = true;
+                if self.flow.is_some() {
```

Comment:
can't this be done cleaner? `if let Some(flow) = self.flow {`?

Reply:
yes